internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/controller/TestHybridStoreRepartitioningWithMultiDataCenter.java

```diff
@@ -394,62 +401,5 @@ public void testRealTimeTopicVersioning() {
             expectedRealTimeTopicName);
       });
     }
-
-    TestWriteUtils.updateStore(storeName, parentControllerClient, new UpdateStoreQueryParams().setPartitionCount(1));
```

Comment:
this was the verifications on the "restored" store. some basic verification are already done in testDeleteAndRecreateStore and all these verifications are also done on new store as part of testEmptyPush and testPartitionCountUpdate, so maybe we can skip them here...